lib/src/components/action_button_component.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+/// [ActionButton] uses for actions
+class ActionButtonComponent extends StatelessWidget {
+  const ActionButtonComponent({
+    Key? key,
+    this.height = 51,
+    required this.title,
+    this.onTap,
+    this.isExpandedCenter = false,
+    required this.icon,
+    this.backgroundColor,
+  }) : super(key: key);
+
+  /// A variable that is used to set the height of the button.
+  final double height;
+
+  /// A variable that is used to set the title of the button.
+  final String title;
+
+  /// A callback for the button.
+  final VoidCallback? onTap;
+
+  /// Used to set the alignment of the button.
+  final bool isExpandedCenter;
+
+  /// A variable that is used to set the icon of the button.
+  final IconData icon;
+
+  /// A variable to set a bacground color for button;
+  final Color? backgroundColor;
+
+  static const _iconSize = 14.0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: InkWell(
+        splashFactory: NoSplash.splashFactory,
```

Comment:
`GestureDetector`?

Reply:
I think about it, we can use MouseRegion and combine it with GestureDetector.